draft-ietf-tls-tls13.md

```diff
@@ -3428,17 +3428,21 @@ mechanism through TLS extensions or some other means.
 
 ## Limits on Key Usage
 
-Using AES-GCM to provide authenticity of authenticated data, content of the plaintext and information leakage (see Note below) protection for the plaintext, the maximum amount of data (padded records) which can be safely encrypted with a single key is 2^48 128-bit blocks (2^64 bytes). When the maximum amount of data is reached, the chance of having a collision among 128-bit blocks of the ciphertext is below 1^(-32) which is negligible. The recommended maximum amount of data (2^64 bytes) is made with the assumption that the (padded) record size is either 128 bits, a multiple of 128 bits or a lot larger than 128 bits. If the (padded) record size is 2^x bytes, then the data limit with GCM for each key is 2^(64-x) records.
+The maximum amount of plaintext data that can be safely encrypted with
+AES-GCM in a session is 2^48 128-bit blocks (2^52 bytes), assuming
+probability of success at 1/2^32. The padding {{record-padding}} is
+considered to be a part of plaintext in this section. If the average
+record size in a session is 2^x bytes, the data limit with AES-GCM
+for each key is 2^(52-x) records.
```

Comment:
Yes, I am rounding this down for a few reasons:

- The use of fractional exponents adds unnecessary complexity
- We begin the analysis with somewhat arbitrary assumption of p=1/2^32 that NIST happen to use [SP 800 38D](http://dx.doi.org/10.6028/NIST.SP.800-38D) . 
- This doesn't consider multi-session collision concerns, e.g. when two AES-128 keys collide

The formula is straighforward: blocks < (128-log2(p))/2.

Because we primarily work in an exponential scale here, 1 bit ~ 100% of error margin. We can tolerate this margin of error because it's widely recognized that rekeying marginally more often doesn't hurt.

Are we OK to keep rounded estimate without additional elaboration?